pkg/cmd/server/origin/master.go

```diff
@@ -776,6 +776,7 @@ func (c *MasterConfig) RunDNSServer() {
 		glog.Fatalf("Could not start DNS: %v", err)
 	}
 	config.DnsAddr = c.Options.DNSConfig.BindAddress
+	config.NoRec = true // do not want to deploy an open resolver
```

Comment:
This looked like a location where in the future we could read more options from c.Options.DNSConfig and set them on the skydns config.